homeassistant/components/vacuum/neato.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, hass, robot):
         self.clean_suspension_charge_count = None
         self.clean_suspension_time = None
 
-    @Throttle(timedelta(seconds=60))
+    @Throttle(timedelta(seconds=300))
```

Comment:
Remove this and set `SCAN_INTERVAL=timedelta(minutes=10)`

Reply:
Thank you for your review :) I have gone ahead and implemented all your requested changes.

Reply:
Also just an FYI for the `vacuum` I set the interval a little bit faster (5 minutes) than the `switch` and `camera` as the vacuum states should update a little faster than 10 minutes.  If you feel I should make it 10 minutes I can definitely change that back.